### ISSUE TITLE
Sort integer keys

### ISSUE THREAD

**Issue by biohazardlover** [dzhibas/SublimePrettyJson#73]
If the sorted keys will look like:
<img width="154" alt="screen shot 2016-08-24 at 11 40 37 am" src="https://cloud.githubusercontent.com/assets/13117408/17917490/a94ca640-69ef-11e6-9de6-997a7f0fe2fa.png">
instead of:
<img width="171" alt="screen shot 2016-08-24 at 11 40 09 am" src="https://cloud.githubusercontent.com/assets/13117408/17917493/b25c9362-69ef-11e6-84df-c7d33beacc21.png">
It will be very useful.

**Comment by sweetw0r** [comment]
+1

**Comment by TheSecEng** [comment]
I agree, the Format & Sort should handle integers better. This will Be investigated.
